recipes/cspice/all/conanfile.py

```diff
@@ -0,0 +1,159 @@
+import os
+import time
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+class CspiceConan(ConanFile):
+    name = "cspice"
+    description = "NASA C SPICE library"
+    license = "TSPA"
+    topics = ("conan", "spice", "naif", "kernels", "space", "nasa", "jpl", "spacecraft", "planet", "robotics")
+    homepage = "https://naif.jpl.nasa.gov/naif/toolkit.html"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "TSPA.txt", "patches/**"]
```

Comment:
```suggestion
    exports_sources = ["CMakeLists.txt", "patches/**"]
    exports = ["TSPA.txt"]
```

TSPA.txt should be exported by `exports` because the license must be available, even when not building the package.

https://docs.conan.io/en/latest/reference/conanfile/attributes.html#exports